xbmc/addons/AddonManager.h

```diff
@@ -279,6 +292,26 @@ namespace ADDON
      */
     bool GetDisabledAddonInfos(std::vector<AddonInfoPtr>& addonInfos, TYPE type);
 
+    /*!
+     * @brief Get a list of disabled add-on's with info's for the on system
+     * available ones with a specific disabled reason.
+     *
+     * @param[out] addonInfos list where finded addon information becomes stored
+     * @param[in] type        The requested type, with "ADDON_UNKNOWN"
+     *                        are all add-on types given back who match the case
+     *                        with value before.
+     *                        If a type id becomes added are only add-ons
+     *                        returned who match them. Default is for all types.
+     * @param[in] disabledReason To get all disabled addons use the value
+     *                           "ADDON_DISABLED_REASON_NONE". If any other value
+     *                           is supplied only addons with that reason will be
+     *                           returned.
+     * @return true if the list contains entries
+     */
+    bool GetDisabledAddonInfos(std::vector<AddonInfoPtr>& addonInfos,
+                               TYPE type,
+                               const ADDON::AddonDisabledReason& disabledReason);
```

Comment:
Only as question, why as new function, instead of?:
```cpp
    bool GetDisabledAddonInfos(std::vector<AddonInfoPtr>& addonInfos, TYPE type, AddonDisabledReason disabledReason = ADDON_DISABLED_REASON_NONE);
```